### DISCUSSION TITLE
Kimai with Nginx behind a Nginx Proxy Manager

### DISCUSSION THREAD

**Discussion by techsupportkoeln** [kimai/kimai#3266] [On-Premise: Installation & Updates] [answered]
Until now I had Kimai running on-premise in an Ubuntu VM with Nginx.
Now i moved Kimai to an off-premise Server behind an Nginx Proxy Manager.

Unfortunately, I always get a 502 error.

Is there any solution in this case? 

Nginx Proxy manager -> Kimai Ubuntu VM with nginx

**Comment by kevinpapst** [comment]
https://www.kimai.org/documentation/webserver-configuration.html#reverse-proxy

**Comment by techsupportkoeln** [comment, marked as answer]
On the VM i had in the nginx a configuration like:

# modern configuration
ssl_protocols TLSv1.3;
ssl_prefer_server_ciphers off;

This was the problem. After we deleted this lines, the nginx proxy works.

thanks to Kevin.
